content/realtime/metachannels.textile

```diff
@@ -115,17 +116,150 @@ metaChannel.subscribe('channel.region.inactive', (msg) => {
 
 While the channel lifecycle metachannel can provide occupancy metrics, there may be a performance impact, since this involves funnelling occupancy updates for every channel in the app to a single metachannel. This approach does not scale well as the number of channels grows, and so has limited use cases. There are two recommended alternatives:
 
 * "Inband channel occupancy events":/realtime/inband-occupancy - for use cases where the entity that needs to determine occupancy is attached to the channel. 
 * Occupancy rules - for use cases where the entity needing to determine channel occupancy is not connected to the channel. A channel occupancy reactor rule allows you to obtain occupancy updates from a set of channels in a centralized way, that doesn't involve attaching to all of them. Please "get in touch":mailto:enterprise-sales@ably.io for further information.
 
-h3(#reactor). Reactor rules
+h2(#sampled-events). Sampled connection and request events
+
+Sampled connection and request events can be used to observe the usage of Ably services. Events are published to metachannels that contain metadata for a sample of connections and API requests. These events can be used to compile statistics on app usage, enabling you to perform arbitrary data processing and aggregation related to client population and client activity.
+
+Note that this is an Enterprise-only feature and must be enabled for each app you would like to sample. Please "contact us":https://ably.com/contact to enable this functionality.
+
+Events are published on the metachannels *@[meta]clientEvents:connections@* and *@[meta]clientEvents:apiRequests@*.
+
+The following code snippet is an example of subscribing to connection events on the @[meta]clientEvents:connections@ channel:
+
+```[javascript]
+const channel = ably.channels.get('[meta]clientEvents:connections');
+
+channel.subscribe('connection.opened', (msg) => {
+    console.log('connection opened: ', msg.data);
+})
+
+channel.subscribe('connection.closed', (msg) => {
+    console.log('connection closed: ', msg.data);
+})
+
+channel.subscribe('connection.failed', (msg) => {
+    console.log('connection failed: ', msg.data);
+})
+
+channel.subscribe('connection.refused', (msg) => {
+    console.log('connection refused: ', msg.data);
+})
+```
+
+h3(#sample). Sample rate
+
+The rate of sampling dictates the number of events published to the metachannels. For example, a rate of 0.1% for connections would publish every 1000th connection event. The sample rate can be configured independently for connections and API requests within the same app.
```

Comment:
> would publish every 1000th connection event

it's not every 1000th event exactly, it's just events at random with a 0.1% probability. "...would publish an average of one message in 1000"

Reply:
Ah, didn't know that. Fixed, thanks.